contrib/utfcpp/template.py

```diff
@@ -0,0 +1,20 @@
+pkgname = "utfcpp"
+pkgver = "4.0.5"
+pkgrel = 0
+build_style = "cmake"
+hostmakedepends = ["cmake", "ninja"]
+pkgdesc = "UTF-8 library"
+maintainer = "Erica Z <zerica@callcc.eu>"
+license = "BSL-1.0"
+url = "https://github.com/nemtrif/utfcpp"
+source = f"{url}/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "ffc668a310e77607d393f3c18b32715f223da1eac4c4d6e0579a11df8e6b59cf"
+
+
+def post_install(self):
+    self.install_license("LICENSE")
+
+
+@subpackage("utfcpp-devel")
```

Comment:
this is a header-only library so the -devel means the parent becomes empty. for these, just don't split them

Reply:
(the reason the parent didn't end up empty and ci didn't catch it is because it installs a /usr/share/utf8cpp/cmake)